Releasenotes.md

```diff
@@ -0,0 +1,77 @@
+# Releasenotes Haal-Centraal-BRP-historie-bevragen
+
+
+## Versie 1.0.0
+
+Alhoewel er nog geen eerdere officiële versie van de Haal-Centraal-BRP-hitorie-bevragen API als release is uitgebracht zijn de specificaties van deze enspoints als onderdeel van de specificatie van Haal-Centraal-BRP-bevragen het afgelopen jaar redelijk stabiel geweest.
+Voor het uitbrengen van release v1.0.0 is besloten om de historie-endpoints in een aparte API onder te brengen en daarvoor een eigen github-repository te maken. Daarbij zijn nieuwe inzichten die het afgelopen jaar zijn opgedaan opgenomen. Dit is een globaal (en niet limitatief) overzicht van wijzigingen die doorgevoerd zijn voordat v1.0.0 is uitgebracht. Enkele van deze wijzigingen zijn breaking ten opzichte van v0.9.0.
+
+We bieden nu ook SDK's aan met gegenereerde [plumbing-code](./code). Daarnaast bieden we een[postman-collectie](./test) t.b.v. testen aan.
+
+### Openapi.yaml :
+
+- server-url is aangepast (omdat het een aparte API is geworden.) --> https://github.com/VNG-Realisatie/Haal-Centraal-BRP-historie-bevragen _**(breaking)**_
```

Comment:
Het is de vraag of het definieren van nationaliteithistorie in een eigen yaml file en opnemen in een eigen repository betekent dat de server-url wordt gewijzigd. Met behulp van een API-Gateway is het heel makkelijk om te routeren van een externe url naar een interne url

Reply:
Het aanpassen van de server-url heb ik op verzoek van @KayodeBakker gedaan. Ik heb zelf geen voorkeur, dus als er een goede overweging is om dit terug te draaien. Misschien even overleg tussen jou en Kayo ?

Reply:
Wat ik mij afvraag, is wat we gaan doen als later de rvig deze API gaat hosten. Gaan we dan de openapi.yaml aanpassen met de nieuwe base url? Gaan we, omdat het nu wordt gemarkeerd als een breaking change, dan ook de major versie nummer ophogen? Of gaan we alleen de getting started document aanpassen en daar aangeven welke base url dan moet worden gebruikt.

Reply:
Goede vragen Melvin. Ik weet het antwoord niet. Ik zal een issue op de Backlog zetten. Lijkt mij een issue dat besproken moet worden op als het RvIG gaat leveren.

Reply:
de vraag of je de base url in de specificaties moet aanpassen is denk ik een heel andere dan de vraag of dat een breaking change is. Ik denk dat het voor nu goed was die base url te wijzigen, maar zie dat meer als documentatie dan specificatie. De base url die is opgenomen is nu immers de url van de demo/testomgeving, niet van een productieomgeving.